src/go/rdctl/pkg/config/config.go

```diff
@@ -51,17 +54,24 @@ var (
 	configDir         string
 	configPath        string
 	defaultConfigPath string
+	configFile        = "rd-engine.json"
+	rdSubDir          = "rancher-desktop"
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
 	var err error
-
-	configDir, err = os.UserConfigDir()
-	if err != nil {
-		log.Fatal("Can't get config-dir: ", err)
+	if runtime.GOOS == "linux" && isWSLDistro() {
+		if defaultConfigPath, err = wslifyConfigPath(); err != nil {
```

Comment:
I think you should have `wslConfigDir()` that only returns the directory. That way you don't need to duplicate the  code that appends `/rancher-desktop/rd-engine.json` in both branches.